proposals/sink_facebook_lead_ads_proposal.md

```diff
@@ -0,0 +1,82 @@
+# Sink Facebook Lead Ads Proposal
+
+## Description
+
+The Facebook Lead Ads Sink Connector send data (from incoming CloudEvents) for ad campaigns, specifying the questions and fields you want to collect from users to create a lead form.
+
+This page describes the design of the Facebook Lead Ads Sink in detail.
+
+## Programming Language
+
+- [x] Golang
+- [ ] Java
+
+## Prerequisites
+
+Before you start you will need the following:
+- The ``ads management`` permission
+- The ``pages manage ads`` permission
+- The ``pages read engagement`` permission
+- The ``pages show list`` permission
+- A Page access token from a User who can perform the ``ADVERTISE`` task on the Page
+
+
+
+## Connector Details
+
+### Configuration
+
+The Facebook Lead Ads Sink needs following configurations to work properly.
+
+| Name        | Required | Default | Description                                                |
+|:------------|:--------:|:-------:|------------------------------------------------------------|
```

Comment:
The configuration should be things like keys. things like questions and name will come inside of the cloud event